src/lucky/route_handler.cr

```diff
@@ -1,15 +1,36 @@
-require "colorize"
-
 class Lucky::RouteHandler
   include HTTP::Handler
 
+  Habitat.create do
+    setting mime_extensions : Hash(String, String) = {} of String => String
+  end
+
   def call(context)
+    original_path = context.request.path
+    format = get_extension(context.request.path)
```

Comment:
Is `format` a reserved crystal word or is github just messing up? If so, maybe use `extension` instead?

Reply:
no, github is just rendering wrong.